js/deleteError.js

```diff
@@ -0,0 +1,10 @@
+const deleteError = (tag) => {
+  tag.classList.remove('input--error');
+  const parent = tag.parentNode;
+  const lastNode = tag.parentNode.lastChild;
+  if(lastNode.className === 'error-message'){
+    parent.removeChild(lastNode);
+  }
```

Comment:
- tag가 뭔지 알 수는 없으니, inputElement로 이름을 수정해서, 의도를 드러낼 수 잇을거같아요.
- 이 부분도 "선택해서 지우는" 노드가 무엇인지, 명시하면 더 좋겠습니다

```suggestion
  const errorMessage = document.querySelector('error-message');
  if(errorMessage) errorMessage.remove()
```